src/actinia_core/resources/mapset_management.py

```diff
@@ -716,3 +720,49 @@ def _execute(self):
             self.lock_interface.unlock(self.target_mapset_lock_id)
             self.finish_message = \
                 "Mapset <%s> successfully unlocked" % self.target_mapset_name
+
+
+class MapsetLockManagementResourceAdmin(ResourceBase):
+    """ Get all locked mapsets
+    """
+    decorators = [log_api_call, check_user_permissions,
+                  very_admin_role, auth.login_required]
+
+    @swagger.doc({
+        'tags': ['Mapset Management'],
+        'description': 'Get all locked mapsets. '
+                       'Minimum required user role: admin.',
+        'responses': {
+            '200': {
+                'description': 'Get a list of locked mapsets ',
+                'schema': LockedMapsetListResponseModel
+            },
+            '500': {
+                'description': 'The error message and a detailed error log',
+                'schema': SimpleResponseModel
+            }
+        }
+    })
+    def get(self):
+        redis_interface = RedisBaseInterface()
```

Comment:
You can use directly the [RedisLockingInterface](https://github.com/mundialis/actinia_core/blob/main/src/actinia_core/common/redis_lock.py#L39) here. As far as I can see, it is also used by the other classes which check for existing mapset locks ([here](https://github.com/mundialis/actinia_core/blob/main/src/actinia_core/resources/mapset_management.py#L659)).